sql/modules/Reconciliation.sql

```diff
@@ -383,10 +391,10 @@ create or replace function reconciliation__add_entry(
 
                 END IF;
         END IF;
-        return 1;
+        return lid;
 
     END;
-$$ language 'plpgsql';
+$$ language 'plpgsql' VOLATILE;
```

Comment:
VOLATILE is the default (https://www.postgresql.org/docs/current/static/xfunc-volatility.html)